.github/workflows/review_cypress_e2e_tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Cypress end-to-end tests
+
+on:
+  pull_request:
+    types: [opened]
```

Comment:
seems is not running because of the ''types: [opened]''

i opened a PR with the same github action and removed that line to see if would run and its running

https://github.com/FightPandemics/FightPandemics/pull/911/files